healpy/projaxes.py

```diff
@@ -865,14 +867,19 @@ def projmap(self, map, nest=False, **kwds):
 #   http://matplotlib.org/examples/pylab_examples/custom_cmap.html
 
 
+
 def get_color_table(
-    vmin, vmax, val, cmap=None, norm=None, badcolor="gray", bgcolor="white"
+    vmin, vmax, val, cmap=None, norm=None, linthresh=1, base=10, linscale=1, badcolor="gray", bgcolor="white"
```

Comment:
I think we need a docstring here to explain `linthresh` and the other parameters. Or is there a better location?